test/minikube/minikube_domain_resync_test.go

```diff
@@ -311,6 +377,43 @@ func TestDomainResync(t *testing.T) {
 	}, 300*time.Second)
 }
 
+func TestLoadBalancerConfigurationResync(t *testing.T) {
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"admin.lbConfig.prefilter":              "not(label(region tiebreaker))",
+			"admin.lbConfig.default":                "random(first(label(node node1) any))",
+			"admin.lbConfig.policies.zone1":         "round_robin(first(label(zone zone1) any))",
+			"admin.lbConfig.policies.nearest":       "random(first(label(pod ${pod:-}) label(node ${node:-}) label(zone ${zone:-}) any))",
+			"admin.lbConfig.fullSync":               "true",
+			"database.sm.resources.requests.cpu":    "0.25",
+			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":    "0.25",
+			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.lbConfig.prefilter":           "not(label(zone DR))",
+			"database.lbConfig.default":             "random(first(label(node ${NODE_NAME:-}) any))",
+			"nuodb.image.tag":                       "4.2-mod",
+		},
+	}
+
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, options, 1, "")
+	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
+	defer testlib.Teardown(testlib.TEARDOWN_DATABASE) // ensure resources allocated in called functions are released when this function exits
+	testlib.StartDatabase(t, namespaceName, admin0, options)
+
+	// Configure one manual policy
```

Comment:
I would add a test for the resync being `false` and verifying that manual configs don't get overridden.

Reply:
Will do.